ingeniamotion/wizard_tests/feedbacks_tests/feedback_test.py

```diff
@@ -114,14 +121,18 @@ def check_symmetry(self, positive, negative):
         return self.check_feedback_tolerance(error, error_msg, self.ResultType.SYMMETRY_ERROR)
 
     @BaseTest.stoppable
-    def check_polarity(self, displacement):
+    def check_polarity(self, displacement: float) -> Polarity:
         self.logger.info("POLARITY CHECK")
         polarity = self.Polarity.NORMAL if displacement > 0 else self.Polarity.REVERSED
         self.logger.info("Feedback polarity detected: %s", polarity.name)
         return polarity
 
     @BaseTest.stoppable
-    def check_resolution(self, displacement):
+    def check_resolution(self, displacement: float) -> ResultType:
+        if not isinstance(self.pair_poles, int):
+            raise TypeError("Pair poles has to be set before resolution checking.")
+        if not isinstance(self.feedback_resolution, int):
+            raise TypeError("Feedbacks has to be set before resolution checking.")
```

Comment:
`Feedback resolution has to be set before resolution checking.`
In addition, the if condition for pair_poles and feedback_resolution should be " is None"